dapp/src/hooks/useAcrePoints.ts

```diff
@@ -53,12 +55,16 @@ export default function useAcrePoints(): UseAcrePointsReturnType {
   const totalBalance = bigIntToUserAmount(data?.claimed ?? 0n, 0)
   const claimableBalance = bigIntToUserAmount(data?.unclaimed ?? 0n, 0)
 
+  const isLoading = userPointsDataQuery.isLoading || pointsDataQuery.isLoading
```

Comment:
The `isLoading` value is used for `Skeleton`.  But I think we don't need to wait for `pointsDataQuery` because the data for this query is already fetched when the user is not yet connected. As I mentioned in another comment, we can use `UserDataSkeleton`

Reply:
Addressed.